lib/i18n/backend/active_record.rb

```diff
@@ -74,6 +76,11 @@ def lookup(locale, key, scope = [], options = {})
             key = key[0..-2]
           end
 
+          if ActiveRecord.config.cache_translations
+            keys = [locale.to_sym] + key.split(I18n::Backend::Flatten::FLATTEN_SEPARATOR).map(&:to_sym)
+            return translations.dig(*keys)
```

Comment:
`return` is not really required here
```suggestion
            
            translations.dig(*keys)
```

Reply:
The return is required actually